core/src/com/unciv/ui/utils/ZoomableScrollPane.kt

```diff
@@ -11,10 +11,6 @@ open class ZoomableScrollPane: ScrollPane(null) {
     var continuousScrollingX = false
 
     init{
-        // Remove the existing inputListener
-        // which defines that mouse scroll = vertical movement
-        val zoomListener = listeners.last { it is InputListener && it !in captureListeners }
```

Comment:
I remember there's a reason this is here. Don't remember what it is, though

Reply:
compile give a warning "**Calling non-final function removeListener in constructor**"